### ISSUE TITLE
[suggestion] create a new extension to truncated files

### ISSUE THREAD

**Issue by DavidBabel** [Destiny1984/XCI-Cutter#5]
everything is in the title

**Comment by Destiny1984** [comment]
Yes, that would help to differentiate trimmed and not trimmed files.

But that extension has to be integrated into SX OS (and other loaders) too.

I will add one when it's actually supported.

**Comment by DavidBabel** [comment]
Or you can set it as a standard, and wait for them to adapt :)

Worst scenario may be to change manually this extension later
